docs/document/content/user-manual/shardingsphere-jdbc/configuration/yaml/mix.cn.md

```diff
@@ -3,4 +3,66 @@ title = "混合规则"
 weight = 6
 +++
 
-TODO
+* 配置项说明
+
+## 配置项说明
+```yml
+rules:
+  - !SHARDING
+    tables:
+      t_order_item_calcite_sharding:
+        actualDataNodes: calcite_ds.t_order_item_calcite_sharding_${0..1}
+        tableStrategy:
+          standard:
+            shardingColumn: item_id
+            shardingAlgorithmName: table_inline_item_id
+      t_user_encrypt_calcite_sharding:
+        actualDataNodes: calcite_ds.t_user_encrypt_calcite_sharding_${0..1}
+        tableStrategy:
+          standard:
+            shardingColumn: user_id
+            shardingAlgorithmName: table_inline_user_id
+    shardingAlgorithms:
+      table_inline_item_id:
+        type: INLINE
+        props:
+          algorithm-expression: t_order_item_calcite_sharding_${item_id % 2}
+      table_inline_user_id:
+        type: INLINE
+        props:
+          algorithm-expression: t_user_encrypt_calcite_sharding_${user_id % 2}
+  - !ENCRYPT
+    encryptors:
+      encryptor_aes:
```

Comment:
In the process of using it, I found that "_" can't be used here, "-" should be used instead. Otherwise, this value will not be recognized, resulting in an error. I configured it in properties format. I don't know if yaml format will have this problem.